sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -3012,14 +3048,53 @@ pi_result cuda_piKernelGetGroupInfo(pi_kernel kernel, pi_device device,
   return PI_INVALID_KERNEL;
 }
 
-/// \TODO Untie from OpenCL
-/// \TODO Not implemented
 pi_result cuda_piKernelGetSubGroupInfo(
     pi_kernel kernel, pi_device device, cl_kernel_sub_group_info param_name,
```

Comment:
```suggestion
    pi_kernel kernel, pi_device device, pi_kernel_sub_group_info param_name,
```
otherwise `\TODO Untie from OpenCL` is still relevant.

Reply:
Ah, I missed that.  Thanks, Alexey.